doc/src/site/sphinx/tutorials/azure_hdi.rst

```diff
@@ -28,7 +28,7 @@ Follow the steps below to create a new H2O Artificial Intelligence for Azure HDI
    .. figure:: ../images/azure_select_h2o_hdinsight.png
       :alt: Select H2O Artificial Intelligence for HDInsight
 
-3. In the next screen, under **Basics**, change the Cluster Type to Spark 2.0.2. Sparkling Water is currently configured to work only on Spark 2.0 and above.
+3. In the next screen, under **Basics**, change the Cluster Type to Spark SUBST_SPARK_VERSION. (Note that Sparkling Water is currently configured to work only on Spark 2.1 and above.)
```

Comment:
The 2.1 can be also subsituted by SUBST_SPARK_MAJOR_VERSION

Reply:
ahh... didn't know that one!